UnityProject/Assets/Scripts/Items/Implants/Organs/Heart.cs

```diff
@@ -96,13 +96,28 @@ public void DoHeartBeat(LivingHealthMasterBase healthMaster)
 	public void Heartbeat(float efficiency)
 	{
 		CirculatorySystemBase circulatorySystem = RelatedPart.HealthMaster.CirculatorySystem;
+		LivingHealthMasterBase HealthBase = RelatedPart.HealthMaster;
 		if (circulatorySystem)
 		{
 			//circulatorySystem.HeartBeat(heartStrength * TotalModified);
 			//TODOH Balance all this stuff, Currently takes an eternity to suffocate
 			// Logger.Log("heart Available  " + ReadyBloodPool);
 			// Logger.Log("heart pumpedReagent " + pumpedReagent);
 
+			//Find out if we have lost limbs and how much blood we'll lose.
+			float TotalBloodLossFromLostLimbs = 0f;
```

Comment:
I feel like this would be better if each body part bled individually instead of the heart bleeding?

Reply:
The heart is not bleeding, the heart is simply a tool that pushes blood out of the body with each heartbeat.

Reply:
certainly. ``Bleeding`` is a state of body parts that makes them leak blood each tick. The state should be handled by the body part itself.

Reply:
done.